### DISCUSSION TITLE
Is anyone not able to run 10000 epoch of training?

### DISCUSSION THREAD

**Discussion by ravsau** [voicepaw/so-vits-svc-fork#749] [Q&A]
My training stops at 1000 epochs on google collab. I have enough credits. Any ideas on how to train to 10k epochs without getting interrupted ?

**Comment by sonphantrung** [comment]
Well, depending on your dataset, unless you have like ~170 hours of dataset, 10k epochs is not needed. You can just interrupt your training session at anytime if you feel that it's enough.

**Comment by Grownz** [comment]
Just edit your `config.json` with higher Epoch-Count: https://github.com/voicepaw/so-vits-svc-fork/discussions/741#discussioncomment-6156082

**Reply by ravsau** [reply]
I have 10000 epochs set in the config file. But my traning stopped at 1760 epoch. 


How can I resume the traning to get to a model trained on 10000 epoch

**Reply by AKhilRaghav0** [reply]
make minor changes in config
